src/components/Button/button.module.scss

```diff
@@ -172,6 +172,12 @@
             background-color: var(--button-counter-active-background-color);
         }
     }
+
+    /** Hides the browser default keyboard focus-visible styles
+        use the ConfigProvider instead */
```

Comment:
nit: this is a separate instruction i think but reads like a run-on sentence

Not sure how it works with css modules, but in my previous codebase sass comments using  "/** */" end up in the outputted css whereas comments using "//" do not.